<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="overlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dcmPath">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DICOM PATH</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dcmPathDisp">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dcmNum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DICOM Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="pnum">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Num</string>
    </property>
   </widget>
   <widget class="QLabel" name="pset">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pnumPrev">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pnumNext">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="psetPrev">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="psetNext">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dcmNumPrev">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dcmNumNext">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="setDICOM">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set DICOM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetDICOM">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset DICOM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level1Prev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level1Next">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level2Next">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level2Prev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level3Next">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level3Prev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level4Next">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>450</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="level4Prev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="setVG">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set VG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delVG">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>610</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete VG</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showDICOM">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show DICOM</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showVG">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show VG</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showWiring">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Wiring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSettingBut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>700</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genData">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>700</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Data</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="savePathDisp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="savePathBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Saving Path</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="genPathDisp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>670</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="genPath">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>700</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="settingFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting file save path</string>
    </property>
   </widget>
   <widget class="QLabel" name="genSave">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>650</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Geneated data save path</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showEvery">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Every Label</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wiringNext">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>580</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="wiringPrev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="wiring">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiring Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="angioFrame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>530</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Angio Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="angioFrameNext">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="angioFramePrev">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dcmOffset">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DICOM Offset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dcmOffsetDown">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>580</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dcmOffsetUp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pnumDisp">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="psetDisp">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level1NumDisp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level2NumDisp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level3NumDisp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level4NumDisp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dcmNumDisp">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>225</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wiringFrameDisp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="angioFrameDisp">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dcmOffsetDisp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="wiringSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="level1Check">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="level2Check">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="level3Check">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="level4Check">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showResult">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>610</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedSaved">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>545</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="setSavedNext">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>541</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="setSavedPrev">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showVGSaved">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Saved</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>dcmPathDisp</tabstop>
  <tabstop>pnumPrev</tabstop>
  <tabstop>pnumNext</tabstop>
  <tabstop>psetPrev</tabstop>
  <tabstop>psetNext</tabstop>
  <tabstop>dcmNumPrev</tabstop>
  <tabstop>dcmNumNext</tabstop>
  <tabstop>setDICOM</tabstop>
  <tabstop>resetDICOM</tabstop>
  <tabstop>level1Prev</tabstop>
  <tabstop>level1Next</tabstop>
  <tabstop>level2Next</tabstop>
  <tabstop>level2Prev</tabstop>
  <tabstop>level3Next</tabstop>
  <tabstop>level3Prev</tabstop>
  <tabstop>level4Next</tabstop>
  <tabstop>level4Prev</tabstop>
  <tabstop>setVG</tabstop>
  <tabstop>delVG</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
